queries.yaml

```diff
@@ -296,12 +296,14 @@ layers:
       - vectordatasource.transform.add_id_to_properties
       - vectordatasource.transform.detect_osm_relation
       - vectordatasource.transform.remove_feature_id
-      - vectordatasource.transform.truncate_min_zoom_to_2dp
+      - vectordatasource.transform.truncate_min_zoom_to_1dp
       - vectordatasource.transform.remap_viewpoint_kinds
       - vectordatasource.transform.unpack_viewpoint_claims
     area-inclusion-threshold: 1
   transit:
     geometry_types: [LineString, MultiLineString, Polygon, MultiPolygon]
+    simplify_start: 5
```

Comment:
Oops, the transit layer wasn't simplified at all!?

Reply:
Zoom 5 because: 

- https://github.com/tilezen/vector-datasource/blob/2eb4486761062f67d473424e6526d1ebf7ba7643/yaml/transit.yaml#L26